gui/tasks/forms.py

```diff
@@ -117,6 +117,15 @@ class RsyncForm(ModelForm):
             " And/Or you do not want validation to be done."
         ),
     )
+    rsync_remotekeyscan = forms.BooleanField(
```

Comment:
Having this as a boolean field is a bad idea can we not just clone the `SSH Keyscan` button from the `Add Replication Task` form?